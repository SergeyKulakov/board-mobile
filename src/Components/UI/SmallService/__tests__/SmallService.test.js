import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { SmallService as Component } from '../index'

describe('SmallService', () => {
  const defaultProps = {
    image: 'image',
    onClick: () => null,
    title: 'title',
    style: undefined,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
