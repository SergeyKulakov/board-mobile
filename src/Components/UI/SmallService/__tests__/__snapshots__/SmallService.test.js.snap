// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallService render snapshot 1`] = `
<Styled(TouchableOpacity)
  onPress={[Function]}
>
  <Styled(View)>
    <Image
      data="image"
      withLoading={true}
    />
  </Styled(View)>
  <Styled(Text)>
    title
  </Styled(Text)>
</Styled(TouchableOpacity)>
`;
